CITATION.cff

```diff
@@ -0,0 +1,41 @@
+cff-version: 1.0.5
+title: A pynxtools plugin for STM (Scanning Tunneling Microscopy) data readers
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Rubel
+    family-names: Mozumder
+    orcid: 'https://orcid.org/0009-0007-5926-6646'
+  - given-names: 	Yichen 
+    family-names: Jin
+    orcid: 'https://orcid.org/0000-0001-9546-1202'
+  - given-names: Florian
+    family-names: Dobener
+    orcid: 'https://orcid.org/0000-0003-1987-6224'
+  - given-names: Lukas
+    family-names: Pielsticker
+    orcid: 'https://orcid.org/0000-0001-9361-8333'
+  - given-names: Sandor
+    family-names: Brockhauser
+    orcid: 'https://orcid.org/0000-0002-9700-4803'
+  - given-names: Carlos-Andres
+    family-names: Palma
+    orcid: 'https://orcid.org/0000-0001-5576-8496'
+doi: 10.5281/zenodo.13799111
```

Comment:
Do we need to write this explicitly?
And if we do so, do we use the doi of the entire zenodo record or the doi for the specific version of the record which becomes important as soon as you have pushed a second version of the code on that same record.
https://help.zenodo.org/docs/deposit/manage-versions/

Reply:
I wrote an email to humpus, how they publish that repo under the `FAIRmat` org. If the project gitHub is included in zenodo, then zenodo is supposed to create a doi. But I see that this doi is included in the example ´CITATION.cff` from humpus. For this mement I will comment it out, but I will keep this point in doto list of the PR, so accidentally it does not marge to the main branch.